webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
```

Comment:
RTCMediaStreamTrackStats doesn't have a reference to RTCRTPStreamStats either - the pointer goes the other way - the reason being that we might have to represent simulcast. See why I find an explicit listing simpler to understand?